templates/impala/search.html

```diff
@@ -3,7 +3,10 @@
   {% set search_placeholder = 'all' %}
 {% endif %}
 {% set search_form = SimpleSearchForm(request, search_cat) %}
+
 <form id="search" action="{{ search_url }}">
+    <div role="status" aria-live="polite" id="announce-suggestions"></div>
+    <label for="search-q" hidden>type to search for add-ons</label>
```

Comment:
The hidden attribute means this won't be visible to assistive tech as well as user agents. In addition the label text needs to be setup to be localized using `{{ _('text here) }}`.